drivers/gles2/shaders/scene.glsl

```diff
@@ -1542,6 +1542,11 @@ FRAGMENT_SHADER_CODE
 
 #ifdef BASE_PASS
 	//none
+
+#ifdef AMBIENT_LIGHT_DISABLED
+    ambient_light = vec3(0.0, 0.0, 0.0);
```

Comment:
Style issue, indentation should use a tab. There's also a typo in the commit log, it should be GL**E**S2.

Reply:
Thank you, I will fix it accordingly.